Makefile

```diff
@@ -33,7 +33,7 @@ ensure: .ensure.phony go.ensure $(SUB_PROJECTS:%=%_ensure)
 	@touch .ensure.phony
 
 .PHONY: build-proto
-PROTO_FILES := $(sort $(wildcard proto/**/*.proto) proto/generate.sh $(wildcard proto/build-container/**/*))
+PROTO_FILES := $(sort $(shell find proto/ -type f -name '*.proto') proto/generate.sh proto/build-container/Dockerfile $(wildcard proto/build-container/scripts/*))
```

Comment:
This is a bit more then needed, but I've done various experiments where I've put folders under proto and these shell finds correctly cover that if we ever actually commit any of those.